examples/bri-3/src/bri/communication/capabilities/createBpiMessage/createBpiMessageCommand.handler.ts

```diff
@@ -0,0 +1,37 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { BpiMessageAgent } from '../../agents/bpiMessages.agent';
+import { BpiMessageStorageAgent } from '../../agents/bpiMessagesStorage.agent';
+import { CreateBpiMessageCommand } from './createBpiMessage.command';
+
+@CommandHandler(CreateBpiMessageCommand)
+export class CreateBpiMessageCommandHandler
+  implements ICommandHandler<CreateBpiMessageCommand>
+{
+  constructor(
+    private readonly agent: BpiMessageAgent,
+    private readonly storageAgent: BpiMessageStorageAgent,
+  ) {}
+
+  async execute(command: CreateBpiMessageCommand) {
+    const { fromBpiSubject, toBpiSubject } =
+      await this.agent.getFromAndToSubjectsAndThrowIfNotExist(
+        command.fromId,
+        command.toId,
+      );
+
+    const newBpiSubjectCandidate = this.agent.createNewBpiMessage(
```

Comment:
bpiSubject in name, replace with message
not important but maybe createNewBpiMessage and other create methods in non-storage agent should also be named something like createXCandidate

Reply:
Fixed var name. I ll do the rename of methods in a separate small PR